examples/pytorch/graphsaint/modules.py

```diff
@@ -3,16 +3,13 @@
 import torch as th
 import dgl.function as fn
 
-
 class GCNLayer(nn.Module):
     def __init__(self, in_dim, out_dim, order=1, act=None,
-                 dropout=0, batch_norm=False, aggr="concat"):
+                 dropout=0, batch_norm=False):
```

Comment:
Do you think it's possible to use our own SAGEConv/GraphConv/etc. with weights?

Reply:
We can replace L48-49 with `GraphConv`, though I'm not sure if it is necessary. This layer also needs to project the input node representations and as a result L25-L36 cannot be fully replaced by `GraphConv`.

Reply:
But L25-36 can be replaced with nn.Linear, nn.BatchNorm, etc., right?  I think `feat_trans` is just a `nn.Sequential` of `Linear`, `ReLU`, and `BatchNorm`.

Side question: `GCNLayer` has some JKNet-like architecture with multiple hops where every hop's output is concatenated at the end, while `GCNNet` stacks multiple `GCNLayer` again.  This looks quite redundant.  Is this what GraphSAINT does?  Moreover, the default uses `1-1-0` architecture so I guess as an example it doesn't have to be that complicated; just showing people how to use `D_norm` should be fine.

Reply:
@BarclayII Yes the model is used by the original GraphSAINT paper [here](https://github.com/GraphSAINT/GraphSAINT/blob/master/graphsaint/pytorch_version/models.py).